packages/neo-one-node-bin/src/bin/neo-one-node.ts

```diff
@@ -0,0 +1,19 @@
+import { FullNode } from '@neo-one/node';
```

Comment:
Need to add process hooks, e.g. https://github.com/neo-one-suite/neo-one/blob/master/packages/neo-one-cli/src/utils/setupCLI.ts#L64.